src/Storage/Collection/LazyCollection.php

```diff
@@ -26,16 +29,50 @@ public function add($element)
     }
 
     /**
-     *  Force loads the proxy objects and returns the real objects
+     * {@inheritdoc}
+     *
+     * Force loads the proxy objects and returns the real objects.
      */
-    public function serialize()
+    public function toArray()
     {
         $output = [];
         foreach ($this as $element) {
+            /** @var Content $proxy */
             $proxy = $element->getProxy();
             $output[] = $proxy->getContenttype() . '/' . $proxy->getSlug();
         }
 
         return $output;
     }
```

Comment:
This should return EntityProxies...or I guess the objects they are proxing..maybe..as that's what every other method on the class works with. 

With the way this is currently coded `get()` could return an object, an EntityProxy, or a string. Huge red flag 👎